gammapy/datasets/map.py

```diff
@@ -1192,6 +1195,10 @@ def from_hdulist(cls, hdulist, name=None, lazy=False, format="gadf"):
             gti = GTI(Table.read(hdulist, hdu="GTI"))
             kwargs["gti"] = gti
 
+        if "META_TABLE" in hdulist:
```

Comment:
Minor comment: maybe rename just to `"META"`? For the other HDU names we don't put the type of the HDU either (like `COUNTS_MAP` or `COUNTS_IMAGE`)

Reply:
I called it `META_TABLE` to have the same name as the class attribute

Reply:
OK, makes sense.